### PR TITLE
FEAT: Refactor tasks

### PR DESCRIPTION
**Pull request by junha-l**
- Remove deleted tasks and re-order tasks

- refactor task_* to use parsed instructions

- add template codes to if - else - on parsed instructions

- clean up comments

### CODE REVIEW

File: pa1.cpp [hunk 1]
```diff
@@ -372,40 +249,40 @@ int main(int argc, char **argv) {
...
-            task_8(fout, argc, argv);
-            break;
-        case 9:
-            task_9(fout);
-            break;
-        case 10:
-            task_10(fout, argc, argv);
+            task_6(fout, instr_seq);
             break;
         case -1:
```
Comment (BH-So) [inline pa1.cpp:267]: `task_num` variable initialize as 0 (at line 233)
  (I think its my fault)
It should be `0` instead of `-1`

File: pa1.cpp [hunk 2]
```diff
@@ -88,269 +74,160 @@ void task_2(ofstream &fout) {
...
+    ///////////////////////////////////////////////////////
+    //////////       Implement From Here      /////////////
+    for (int i=0; i<instr_seq->length; i++) {
+        string command = instr_seq->instructions[i].command;
+        if (command.compare("append") == 0) {
+            /* Implement */
+        } else if(command.compare("insert_at") == 0) {
+            /* Implement */
+        } else {
+            /* Implement */
```
Comment (BH-So) [inline pa1.cpp:106]: There is no description how to handle error
How about fill this line as below
```
cerr << "Invalid command" << endl;
exit(-1);
```

File: pa1.cpp [hunk 3]
```diff
@@ -88,269 +74,160 @@ void task_2(ofstream &fout) {
...
+    string answer;
+    
+    ///////////////////////////////////////////////////////
+    //////////       Implement From Here      /////////////
+    for (int i=0; i<instr_seq->length; i++) {
+        string command = instr_seq->instructions[i].command;
+        if (command.compare("push") == 0) {
+            /* Implement */
+        } else {
+            /* Implement */
```
Comment (BH-So) [inline pa1.cpp:139]: Same comments for here

File: pa1.cpp (hunk 2 above)
Comment (junha-l) [reply pa1.cpp:106]: yeah. That sounds better

File: pa1.cpp (hunk 1 above)
Comment (junha-l) [reply pa1.cpp:267]: OK. I'll change it

**Pull request merged**